tdm-be/src/lib/cron.ts

```diff
@@ -0,0 +1,66 @@
+import environment from '~/lib/config';
+import { customFile, filesLocation } from '~/lib/files';
+import { cronLogger } from '~/lib/logger';
+
+import cron from 'node-cron';
+
+import fs from 'node:fs/promises';
+
+const createFileCron = async (directory: string) => {
```

Comment:
<!-- Quibbles are very much like nitpick:, except it does not conjure up images of lice and animal hygiene practices.  -->
**quibble (non-blocking):** ‏The function's name is confusing.

This *create*FileCron, in fact, does delete files.
In itself, it has almost nothing to do with cron.
I'd rather name it `deleteOldFiles`